include/openssl/e_os2.h

```diff
@@ -252,6 +252,15 @@ typedef unsigned __int64 uint64_t;
 #  include <stdint.h>
 #  undef OPENSSL_NO_STDINT_H
 # endif
+# if defined(__STDC_VERSION__) && __STDC_VERSION__ >= 199901L && \
```

Comment:
This seems like a less intuitive place to put this than where it came from.

It's not a breaking change, but still...

Reply:
I disagree.  This is where we define all the other int and uint types, so why we define our variant of the intmax and uintmax types somewhere else is incomprehensible to me.

Reply:
Perhaps the rest should be moved....

Reply:
> _We've always done it that way_ is never an excuse.

I think that having *one* "architecture fixup" definition separate from all the others is not very consistent.  Furthermore, it's quite obviously confusing, even to us who work with the code every day.